ovos_plugin_manager/language.py

```diff
@@ -116,37 +118,41 @@ class OVOSLangDetectionFactory:
 
     @staticmethod
     def create(config=None):
-        """Factory method to create a LangDetection engine based on configuration.
+        """
+        Factory method to create a LangDetection engine based on configuration
 
-        The configuration file ``mycroft.conf`` contains a ``language`` section with
+        The configuration file `mycroft.conf` contains a `language` section with
         the name of a LangDetection module to be read by this method.
 
-         "language": {
+        "language": {
             "detection_module": <engine_name>
-          }
+        }
         """
+        config = config or Configuration()
+        if "language" in config:
+            config = config["language"]
+        lang_module = config.get("detection_module",
+                                 "libretranslate_detection_plug")
```

Comment:
default value should now be from ovos-classifiers  (ovos-core dep already)

https://github.com/OpenVoiceOS/ovos-config/blob/dev/ovos_config/mycroft.conf#L643

Reply:
Updated fallback handling with a note that `language` is handled differently compared to other factory methods